doc/api_workflows/field_reference/participant_summary_field_list.rst

```diff
@@ -121,47 +124,77 @@ Participant Summary Field List
+      indicates the time at which the participant completed consent, regardless of when it was sent to RDR.
+
+  consentForDvElectronicHealthRecordsSharingAuthored
+      indicates the time at which the participant completed consent, regardless of when it was sent to RDR.
+
+  consentForCABoRTimeAuthored
+      indicates the time at which the participant completed California Bill of Rights consent, regardless of when it was sent to RDR
+
   questionnaireOnOverallHealth
       indicates status for Overall Health PPI module.
 
   questionnaireOnOverallHealthTime
-      A questionnaire on OverallHealthTime that a participant can fill out.
+      indicates the time at which the RDR received notice of overall health questionnaire.
+
+  questionnaireOnOverallHealthAuthored
+      indicates the time at which the participant completed the overall health questionnaire.
 
   questionnaireOnHealthcareAccess
       A questionnaire on HealthcareAccess that a participant can fill out.
 
   questionnaireOnHealthcareAccessTime
-      A questionnaire on HealthcareAccessTime that a participant can fill out.
+      indicates the time at which the RDR received notice of health care access questionnaire.
+
+  questionnaireOnHealthcareAccessAuthored
+      indicates the time at which the participant completed the health care access questionnaire.
 
   questionnaireOnlifestyle
       A questionnaire on lifestyle that a participant can fill out.
 
   questionnaireOnlifestyleTime
-      A questionnaire on lifestyleTime that a participant can fill out.
+      indicates the time at which the RDR received notice of lifestyle questionnaire.
+
+  questionnaireOnlifestyleAuthored
+      indicates the time at which the participant completed the lifestyle questionnaire.
 
   questionnaireOnMedicalHistory
       A questionnaire on MedicalHistory that a participant can fill out.
 
   questionnaireOnMedicalHistoryTime
-      A questionnaire on MedicalHistoryTime that a participant can fill out.
+      indicates the time at which the RDR received notice of medical history questionnaire.
+
+  questionnaireOnMedicalHistoryAuthored
+      indicates the time at which the participant completed the medical history questionnaire.
 
   questionnaireOnMedications
       A questionnaire on Medications that a participant can fill out.
 
   questionnaireOnMedicationsTime
-      A questionnaire on MedicationsTime that a participant can fill out.
+      Indicates the time at which the RDR received notice of medications questionnaire.
+
+  questionnaireOnMedicationsAuthored
+      indicates the time at which the participant completed the medications questionnaire.
 
   questionnaireOnFamilyHealth
       A questionnaire on FamilyHealth that a participant can fill out.
 
   questionnaireOnFamilyHealthTime
-      A questionnaire on FamilyHealthTime that a participant can fill out.
+      Indicates the time at which the participant completed the family health questionnaire.
```

Comment:
Is this the time the RDR received notice of the family health questionnaire?

Reply:
yes, done.